### PR TITLE
Float testing

### PR DESCRIPTION
**Pull request by tudorcebere**
## Description
Better testing and better support for floats.

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

**Review by iamtrask — APPROVED** [review]
Very nice! Clean and readable!

File: src/syft/lib/python/float.py [hunk 1]
```diff
@@ -78,99 +81,224 @@ def __repr__(self) -> str:
...
         return PrimitiveFactory.generate_primitive(value=res)
 
     @syft_decorator(typechecking=True, prohibit_args=False)
-    def __radd__(self, other: Any) -> PyPrimitive:
-        return Float(value=other)
+    def __abs__(self) -> SyPrimitiveRet:
+        res = super().__abs__()
+        return PrimitiveFactory.generate_primitive(value=res)
+
+    @syft_decorator(typechecking=True, prohibit_args=False)
```
Comment (gmuraru) [inline src/syft/lib/python/float.py:93]: Q: Can we put ```True```? (for ```prohibit_args```)

File: src/syft/lib/python/float.py [hunk 2]
```diff
@@ -78,99 +81,224 @@ def __repr__(self) -> str:
...
+            return NotImplemented
+
+        r, q = res
+
+        return (
+            PrimitiveFactory.generate_primitive(value=r),
+            PrimitiveFactory.generate_primitive(value=q),
+        )
+
+    @syft_decorator(typechecking=True, prohibit_args=False)
```
Comment (gmuraru) [inline src/syft/lib/python/float.py:187]: Q: Also for the other functions, maybe it is ok to add ```prohibit_args=True```

**Review by gmuraru — APPROVED** [review]
LGTM!

**Pull request merged**